### PR TITLE
Update maintainer_merge_review.yml

### PR DESCRIPTION
**Pull request by bryangingechen**
testing again... 

---

maybe the bot only responds to review comments work on unmerged PRs?

### CODE REVIEW

**Review by bryangingechen — COMMENTED** [review]
maintainer merge

File: .github/workflows/maintainer_merge_review.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Maintainer merge (review)
 
 on:
   pull_request_review_comment:
```
Comment (bryangingechen) [inline .github/workflows/maintainer_merge_review.yml]:
```suggestion
  pull_request_review:
```
I'm going to switch this file to pull_request_review and possibly add a new file for pull_request_review_comment.

**Review by bryangingechen — COMMENTED** [review]
maintainer merge

**Review by jcommelin — COMMENTED** [review]
extra text

maintainer merge

**Pull request merged**